pshtt.py

```diff
@@ -168,7 +173,7 @@ def basic_check(endpoint):
         return
 
     # Endpoint is live, analyze the response.
-    endpoint.headers = dict(req.headers)
+    endpoint.headers = req.headers
```

Comment:
I'm now storing the original `requests.structures.CaseInsensitiveDict` form of the `headers` object on the `Endpoint` model rather than a converted `dict`. This is to make HSTS header extraction case-insensitive, at least under #12 is done and a proper header and HSTS parser is in place.